bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -1939,4 +1941,27 @@ public ServerConfiguration setNettyMaxFrameSizeBytes(int maxSize) {
         super.setNettyMaxFrameSizeBytes(maxSize);
         return this;
     }
+
+    /**
+     * returns whether it is allowed to have multiple ledger/index Directories in the same 
+     * filesystem partition
+     *
+     * @return  
+     */
+    public boolean isAllowMultipleDirsUnderSamePartition() {
```

Comment:
"Partition" is a very confusing name. I would prefer using "disk" rather than "partition".